variables.tf

```diff
@@ -80,6 +104,18 @@ variable "kms_encryption_enabled" {
   default     = false
 }
 
+variable "skip_en_cos_auth_policy" {
+  type        = bool
+  description = "Set to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group to interact with your Cloud Object Storage instance. No policy is created if `var.cos_integration_enabled` is set to false."
```

Comment:
This description is now out of date. The auth policy is scoped to the exact EN instance and bucket